src/components/ADempiere/Form/VPOS/ConfirmDelivery/index.vue

```diff
@@ -445,6 +445,16 @@ export default {
             showClose: true
           })
         }
+        const suggestionOpen = results.length
+        if (this.isEmptyValue(queryString) || queryString.length < 4) {
+          // not show list
+          callBack(results)
+          return
+        }
+        if (suggestionOpen <= 1) {
```

Comment:
This must be comparable with 1 instead, is a search is void then what is the reason for getting back the product info?

Reply:
HI @yamelsenih . If the search is null the list collapses and throws an error message